.github/workflows/lesson_checks.yml

```diff
@@ -0,0 +1,68 @@
+# Contains jobs corresponding to verify Oppia lesson compatibility.
+
+name: Checks for Oppia Lesson Compatibility
+
+on:
+  workflow_dispatch:
```

Comment:
This means it can be triggered manually, right? If so, that's great -- just wanted to make sure we have that functionality since I think there will be times where we'd want to generate a report on demand.

Reply:
Yup, that's exactly what this trigger is used for.

Reply:
Nothing more to do here, so resolving.